### PR TITLE
Feature/add string art

### PR DESCRIPTION
**Pull request by jdmedlock**
Add string art application specification

### CODE REVIEW

File: Projects/String-Art.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Application Name
```
Comment (florinpop17) [inline Projects/String-Art.md]: Could you add the Application name?

File: Projects/String-Art.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+trajectory - the incremental distance from the previous line based on the
+endpoints.
+-   [ ] When either endpoint of the line touches the boundary of the enclosing
+window change it's direction and randomly alter its angle.
+-   [ ] Progressively fade the intensity of old lines so that only the most
+recent 10-20 lines are visible to create the sense of movement or "ripple".
+
+## Bonus features
+
+-   [ ] Allow the user to specify the length of the line and it's velocity
```
Comment (florinpop17) [inline Projects/String-Art.md]: Could we keep the "User can, user..." format? Just an observation, maybe in the future we won't be able to keep this format 😅

**Comment by florinpop17** [comment]
Also @jdmedlock, this project might be a little "harder" (at least to comprehend) to add at this stage of the collection. We could keep it in backlog and add it later. What do you say?

File: Projects/String-Art.md (hunk 1 above)
Comment (jdmedlock) [reply Projects/String-Art.md]: Added application name to section title

File: Projects/String-Art.md (hunk 2 above)
Comment (jdmedlock) [reply Projects/String-Art.md]: Updated user stories. Good catch! Keeping a uniform format for stories will make it easier on the users.

**Pull request merged**